helix-term/src/commands.rs

```diff
@@ -308,8 +308,7 @@ impl MappableCommand {
         goto_line_end, "Goto line end",
         goto_next_buffer, "Goto next buffer",
         goto_previous_buffer, "Goto previous buffer",
-        // TODO: different description ?
-        goto_line_end_newline, "Goto line end",
+        goto_line_end_newline, "Goto line end and add a new line",
```

Comment:
This is not the correct behavior though -- it doesn't _add_ a newline, it simply places the cursor on the newline. `goto_line_end` places it on the last character instead.

Reply:
You're right, will fix.